deepedit/ignite/README.md

```diff
@@ -70,6 +70,14 @@ python ./train.py
     --output      deepedit_model/ \
     --epochs      100
 
+# Train a DeepEdit model using multi gpu
+python -m torch.distributed.launch \
```

Comment:
Maybe better to use the new PyTorch API?
https://github.com/pytorch/pytorch/blob/master/torch/distributed/run.py

Thanks.

Reply:
Agree